test/views/DepartureSummaryViewSpec.scala

```diff
@@ -26,10 +29,15 @@ import views.html.summary.departure_summary_page
 
 class DepartureSummaryViewSpec extends UnitViewSpec with Stubs with Injector {
 
-  val cachedData = cacheMapData("EDL")
+  val cachedDataOutOfUk = cacheMapData("EDL")
+  val cachedDataBackIntoUk = cachedDataOutOfUk + (GoodsDeparted.formId -> Json.toJson(
```

Comment:
IMO naming, `cachedDataBackIntoUk` that use inside `cachedDataOutOfUk` I'll use this `cacheMapData("EDL")` once again